broadinstitute/orsp-pub
# BTRX - 816 - Required Field Fix

## Addresses
[BTRX-816](https://broadinstitute.atlassian.net/browse/BTRX-816): ORSP: Required field message for "Broad study activities" not working as expected

## Changes
- There were some warning variable flags missing in `CancelEdits`

## Testing
1. Log in with valid credentials
2. Go to a project
3. Click on "Edit Information"
4. Delete the "Broad study activities" content
5. Click on "Submit Edits"
6. Click "Cancel"
---
